### ISSUE TITLE
Dialog before deleting the complete drawing 

### ISSUE THREAD

**Issue by HoWilgh** [szafir1100/notepad#18]
An additional dialog after pressing the waste bin button in the "draw options" dialog is necessary to warn the user that complete drawing will be deleted.

**Comment by tmarzeion** [comment]
@HoWilgh You're totally right, I can imagine how many people said "FCK ME" after clicking that button :)
